cmd/edenConfig.go

```diff
@@ -0,0 +1,307 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/lf-edge/eden/pkg/utils"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	defaultEveRepo    = "https://github.com/lf-edge/eve.git"
```

Comment:
I see a lot of these types of const scattered through the codebase here. Wouldn't it be better to have a single `const.go` with all of them in a single place (and also all of them having comments explaining what they are)

Reply:
Done:
https://github.com/lf-edge/eden/blob/ea676f293e39ed98b8fd87103d1a4880316554e5/cmd/defaults.go#L3-L21